engine/backends/fstar/fstar_backend.ml

```diff
@@ -692,10 +692,10 @@ struct
   and pquote span { contents; _ } =
     List.map
       ~f:(function
-        | `Verbatim code -> code
-        | `Expr e -> pexpr e |> term_to_string
-        | `Pat p -> ppat p |> pat_to_string
-        | `Typ p -> pty span p |> term_to_string)
+        | Verbatim code -> code
+        | Expr e -> pexpr e |> term_to_string
+        | Pattern p -> ppat p |> pat_to_string
+        | Typ p -> pty span p |> term_to_string)
```

Comment:
Generally, the use of explicit constructors is better so this change seem fine, unless we were relying on subtyping or on one of these constructor names also being used in another type. As long as the code compiles and typechecks this should not be the case.